### PR TITLE
Changelog: bump to v1.0.3

### PR DESCRIPTION
**Pull request by ericsyh**
### What problem does this PR solve? 
Prepare for v1.0.3 release

### What is changed and how it works?
Add v1.0.3 release notes in Changelog

### Check List 

Tests 

 - No code

Related changes

 - Need to be included in the release note

### CODE REVIEW

**Comment by ericsyh** [comment]
@WangXiangUSTC @csuzhangxc PTAL

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,38 @@
 
 All notable changes to this project will be documented in this file.
 
+## [1.0.3] 2019-12-13
+
+### Improvements
+
+- Add the command mode in dmctl
+- Support replicating the `AlTER DATABASE` DDL statement
```
Comment (csuzhangxc) [inline CHANGELOG.md]:
```suggestion
- Support replicating the `ALTER DATABASE` DDL statement
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,38 @@
...
+- Optimize the error message output
+
+### Bug fixes
+
+- Fix the panic-causing data race issue occurred when the full import unit pauses or exits
```
Comment (csuzhangxc) [inline CHANGELOG.md]: a little strange? 🤔 the same with L31.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,38 @@
...
+
+- When upgrading from a previous version, note that you must upgrade all DM components (dmctl/DM-master/DM-worker) together
+
+### Detailed Bug Fixes and Changes
+
+- Add the command mode in dmctl [#364](https://github.com/pingcap/dm/pull/364)
+- Optimize the error message output [#351](https://github.com/pingcap/dm/pull/351)
+- Optimize the output of the `query-status` command [#357](https://github.com/pingcap/dm/pull/357)
+- Optimize the privilege check for different task modes [#374](https://github.com/pingcap/dm/pull/374)
+- Support checking the duplicate route-rules or filter-rules [#385](https://github.com/pingcap/dm/pull/385)
```
Comment (csuzhangxc) [inline CHANGELOG.md]: not accurate for `支持对重复引用的路由配置和过滤配置进行检查`?

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,38 @@
...
+- Support replicating the `AlTER DATABASE` DDL statement [#389](https://github.com/pingcap/dm/pull/389)
```
Comment (csuzhangxc) [inline CHANGELOG.md]:
```suggestion
- Support replicating the `ALTER DATABASE` DDL statement [#389](https://github.com/pingcap/dm/pull/389)
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,38 @@
...
+- Optimize the retry mechanism for anomalies [#391](https://github.com/pingcap/dm/pull/391)
+- Fix the panic-causing data race issue occurred when the full import unit pauses or exits [#353](https://github.com/pingcap/dm/pull/353)
+- Fix the issue that `stop-task` and `pause-task` might not take effect when retrying SQL operations to the downstream [#377](https://github.com/pingcap/dm/pull/377)
```
Comment (csuzhangxc) [inline CHANGELOG.md]: use `https://github.com/pingcap/dm/pull/400` instead of #377?

File: CHANGELOG.md [hunk 6]
```diff
@@ -2,6 +2,38 @@
...
+- Upgrade golang to v1.13 and upgrade the version of other dependencies [#362](https://github.com/pingcap/dm/pull/362)
+- Filter the error that the context is canceled when a SQL statement is being executed [#382](https://github.com/pingcap/dm/pull/382)
+- Fix the issue that the error occurred when performing a rolling upgrade to DM monitor using DM-ansible causes the upgrade to fail [#408](https://github.com/pingcap/dm/pull/408)
```
Comment (csuzhangxc) [inline CHANGELOG.md]: Is `update` better than `upgrade` here?

Comment (ericsyh) [reply CHANGELOG.md]: L35?
Yep, it should be `rolling update` rather than `upgrade`

File: CHANGELOG.md (hunk 5 above)
Comment (ericsyh) [reply CHANGELOG.md]: I will replace it.

File: CHANGELOG.md (hunk 3 above)
Comment (ericsyh) [reply CHANGELOG.md]: Support checking the duplicate configs in task route-rules or filter-rule ?

File: CHANGELOG.md (hunk 2 above)
Comment (ericsyh) [reply CHANGELOG.md]: Fix the panic issue during the import unit pauses or exits because of the data race?

Comment (csuzhangxc) [reply CHANGELOG.md]: Is `during` suitable for `pauses` and `exists`? maybe you can ask for i18n team. 😢

File: CHANGELOG.md (hunk 3 above)
Comment (csuzhangxc) [reply CHANGELOG.md]: 这里强调的是同一条 rule 被重复引用多次，现在不能检查不同 rule 包含同样的内容。

**Comment by csuzhangxc** [comment]
@WangXiangUSTC PTAL

**Pull request merged**